packages/graphql/src/schema/validation/utils/map-error.test.ts

```diff
@@ -0,0 +1,311 @@
+            });
+            test("UserAuthorizationFilterRule", () => {
+                const error = new GraphQLError(
+                    'Invalid argument: filter, error: Field "nested" of required type "UserAuthorizationFilterRule" was not provided.',
+                    {
+                        nodes: [argumentNode, directiveNode],
+                        extensions: { exception: { code: VALIDATION_ERROR_CODES.AUTHORIZATION } },
+                    }
+                );
+                const mappedError = mapError(error);
+                expect(mappedError).toHaveProperty("message");
+                expect(mappedError.message).toBe(
+                    'Invalid argument: filter, error: Field "nested" of required type was not provided.'
+                );
+            });
+            test("UserAuthorizationWhere", () => {
+                const error = new GraphQLError(
+                    'Invalid argument: filter, error: Field "nested" of required type "UserAuthorizationWhere" was not provided.',
+                    {
+                        nodes: [argumentNode, directiveNode],
+                        extensions: { exception: { code: VALIDATION_ERROR_CODES.AUTHORIZATION } },
+                    }
+                );
+                const mappedError = mapError(error);
+                expect(mappedError).toHaveProperty("message");
+                expect(mappedError.message).toBe(
+                    'Invalid argument: filter, error: Field "nested" of required type was not provided.'
+                );
+            });
+        });
+
+        describe("expected type to be an object", () => {
+            test("JWTPayloadWhere", () => {
+                const error = new GraphQLError(
+                    'Invalid argument: filter, error: Expected type "JWTPayloadWhere" to be an object.',
+                    {
+                        nodes: [argumentNode, directiveNode],
+                        extensions: { exception: { code: VALIDATION_ERROR_CODES.AUTHORIZATION } },
+                    }
+                );
+                const mappedError = mapError(error);
+                expect(mappedError).toHaveProperty("message");
+                expect(mappedError.message).toBe("Invalid argument: filter, error: Expected type to be an object.");
+            });
+            test("UserWhere", () => {
+                const error = new GraphQLError(
+                    'Invalid argument: filter, error: Expected type "UserWhere" to be an object.',
+                    {
+                        nodes: [argumentNode, directiveNode],
+                        extensions: { exception: { code: VALIDATION_ERROR_CODES.AUTHORIZATION } },
+                    }
+                );
+                const mappedError = mapError(error);
+                expect(mappedError).toHaveProperty("message");
+                expect(mappedError.message).toBe("Invalid argument: filter, error: Expected type to be an object.");
+            });
```

Comment:
Still not sure how I feel about types which will eventually exist in the schema...